docker-compose.dev.yml

```diff
@@ -42,24 +42,6 @@ services:
     networks:
       - esnet
 
-  webapp:
```

Comment:
If we are running the second command from the README, we don't have to have the webapp running as part of docker-compose